iphone/Classes/TiUIView.m

```diff
@@ -605,6 +605,7 @@ - (void)setBackgroundImage_:(id)image
   if (backgroundRepeat) {
     [self renderRepeatedBackground:bgImage];
   } else {
+    RELEASE_TO_NIL([self backgroundImageLayer].contents);
```

Comment:
This line is causing crash. If I remove this line, JSGarbageCollect is working nice. I'll do more testing.